content/docs/studio/user-guide/model-registry/use-models.md

```diff
@@ -1,17 +1,35 @@
 # Use models
 
 Whether you need to download your models to use them, or you're looking to set
-up some automation in CI/CD to deploy them, Iterative Studio provides these
+up some automation in CI/CD to deploy them, DVC Studio provides these
 capabilities.
 
 ## Download models
 
 If your model file is DVC-tracked, you can download any of its registered
-versions using Studio [REST API], `dvc get`, or DVC [Python API].
+versions with DVC Studio from the web interface or in your code.
 
-### Using Studio API
+Prerequisites:
 
-Using Studio API will require you to add your remote storage secrets to
+- Model stored with DVC with S3 or Azure [remote].
```

Comment:
As you pointed out elsewhere @jellebouwman, I guess we should also include http here and in https://dvc.org/doc/studio/rest-api?